src/pages/practice/innersource.md

```diff
@@ -0,0 +1,56 @@
+---
+templateKey: practice-page
+title: InnerSource
+subtitle: Re-use code, collaborate, and use open practices inside your organisation.
+date: 2024-05-31T08:11:26.394Z
+authors:
+  - kattekrab
+  - rmarting
+tags:
+  - culture
+mobiusTag: foundation
+icon: /images/innersource-for-dummies.jpeg
+whatIs: I﻿nnerSource is an approach to software development inside an
+  organisation that is inspired by the success of the Open Source Software
+  movement.  A diverse community of practitioners is sharing and refining
+  [patterns at InnerSource commons](https://patterns.innersourcecommons.org/)
+whyDo: I﻿nnerSource promotes collaboration, and code-reuse. People say it's a
+  useful way to share knowledge, improve code quality, and increase the reach
+  and impact of their work.
+howTo: >-
+  T﻿o get started, make it easy for people outside your team to see your source
+  code, and share a guide on how to contribute.
+
+
+  * [Start as an experiment](https://patterns.innersourcecommons.org/p/start-as-experiment)
+
+  * A﻿gree to, and publish standards 
+
+  * U﻿se asynchronous communication tools
+
+  * G﻿rant access to source code repository
+
+  * Make d﻿ocumentation available
```

Comment:
Maybe linking to [InnerSource Portal](https://patterns.innersourcecommons.org/p/innersource-portal) pattern

Reply:
✅